crates/host/src/bindle.rs

```diff
@@ -36,12 +35,36 @@ const KEYRING_FILE: &str = "keyring.toml";
 /// Authentication method
 #[derive(Clone)]
 pub enum Auth {
-    /// No authentication
-    None(NoToken),
     /// HTTP authentication
     Http(HttpBasic),
     /// Token authentication
     LongLived(LongLivedToken),
+    /// No authentication
+    None(NoToken),
+}
+
+impl Default for Auth {
```

Comment:
Small nit: Probably better to derive default and then just use the `#[default]` attr to pick this one

Reply:
That was not possible with the existing code, but I refactored the enum to allow for this